backend/backend/views.py

```diff
@@ -1,6 +1,4 @@
 from django.conf import settings
-from django.core.exceptions import ObjectDoesNotExist
```

Comment:
Just curious, any particular reason to repackage the Django exception in our own?

Reply:
I'm not sure I understand the question. Do you mean replacing Django's `ObjectDoesNotExist` with a new `GoneException`?

If so, I hadn't thought about using Django builtins (just looked at DRF's exceptions, and there were none for 410), but I'd expect throwing `ObjectDoesNotExist` to result in a 404 or 500.